x-pack/plugins/siem/public/management/pages/policy/view/policy_list.tsx

```diff
@@ -174,6 +233,7 @@ export const PolicyList = React.memo(() => {
         pagination={paginationSetup}
         onChange={handleTableChange}
         data-test-subj="policyTable"
+        hasActions={false}
```

Comment:
I think this should be `true` no?  I think this prop impacts how the actions are handled in Mobile view.

Reply:
I'm going off of the example in the docs.  There, they set `hasActions` to `false` if they are using any custom actions, which I am.  I was getting error when I had set this to `true`, but I was not following the format of actions, instead using `render()` for my custom set.